src/run/run.c

```diff
@@ -1761,14 +1759,16 @@ static int start_transient_service(sd_bus *bus) {
                                 return log_error_errno(slave, "Failed to open pty slave: %m");
 
                 } else if (arg_transport == BUS_TRANSPORT_MACHINE) {
-                        _cleanup_(sd_bus_unrefp) sd_bus *system_bus = NULL;
+                        _cleanup_(sd_bus_flush_close_unrefp) sd_bus *system_bus = NULL;
```

Comment:
Hmm, should we close the default system bus here? It could still be used by later calls?

Reply:
Hmm, indeed. I will drop the change.